types/require-dir/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for require-dir 0.3
+// Project: https://github.com/aseemk/requireDir
+// Definitions by: My Self <https://github.com/me>
```

Comment:
Could you replace `My Self <https://github.com/me>` with real information? Thanks 🐱

Reply:
Sure! I think this can be taken from Git by `dts-gen` automatically.